api/base/versioning.py

```diff
@@ -14,6 +14,8 @@
 # Note that this version will not deprecate snake_case yet.
 KEBAB_CASE_VERSION = '2.18'
 
+CREATE_REGISTRATION_FIELD_CHANGE_VERSION = '2.19'
```

Comment:
update relevant tests to use 2.19

Reply:
ticket for updating docs?

Reply:
it is a `TODONE`, as [there is a ticket](https://openscience.atlassian.net/browse/ENG-1170)

Reply:
it is a `TODONE`, as [there is a ticket](https://openscience.atlassian.net/browse/ENG-1170)